autospec/buildreq.py

```diff
@@ -560,3 +585,4 @@ def load_specfile(specfile):
     specfile.buildreqs = buildreqs
     specfile.requires = requires
     specfile.cargo_bin = cargo_bin
+    specfile.extra_cmake = " ".join(extra_cmake)
```

Comment:
It looks like this overwrites the `extra_cmake` args specified via the `cmake_args` configuration file (https://github.com/clearlinux/autospec/blob/master/autospec/config.py#L672).

Why not just append like so:
```
specfile.extra_cmake += " " + " ".join(extra_cmake)
```
so user-set cmake args aren't being overwritten.

Reply:
done

Reply:
The change still won't add a new space between the previous `extra_cmake` and the new `extra_cmake`, so potentially you would have `-USER -FLAGS-CATKIN -FLAGS` when you should have a space between `-FLAGS` and `-CATKIN`.